microprofile/openapi/src/main/java/io/helidon/microprofile/openapi/MPOpenAPIBuilder.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.microprofile.openapi;
+
+import java.util.Optional;
+
+import io.helidon.openapi.OpenAPISupport;
+
+import io.smallrye.openapi.api.OpenApiConfig;
+import org.jboss.jandex.IndexView;
+
+/**
+ * Fluent builder for OpenAPISupport in Helidon MP.
+ */
+public final class MPOpenAPIBuilder extends OpenAPISupport.Builder {
+
+    private Optional<OpenApiConfig> openAPIConfig;
+    private Optional<IndexView> indexView;
+
+    @Override
+    public OpenApiConfig openAPIConfig() {
+        return openAPIConfig.get();
```

Comment:
May not be reading things clearly but `openAPIConfig` can be `null` here, right?

Reply:
Yes. In practice, our own code will have invoked the setters before the getters.